content/getting-started/registering.md

```diff
@@ -6,6 +6,42 @@ next: "/getting-started/setting-up/"
 
 # Registering
 
-<!--
-TODO Briefly explain how to create an account and link to https://www.codewars.com
--->
+So you decided to join? That's great! There's only a couple of steps to create a new account and start solving tasks, learn, and improve your skills!
+
+## First step
+
+First, you need to go to [Sign Up](https://www.codewars.com/join) page, if you haven't yet. You will be presented with a board of many languages to choose from:
+
+![languages board](./img/registering_01_choose_language.png)
+
+Choose whatever language you like, want to learn, or are familiar with. Here you can pick any language available, and it will be used to present you your very first, introductory challenge.
+
+## Your first challenge
+
+It's an entry level task which you need to solve to prove your dedication and commitment to learning. Let's see...
+
+![first task](./img/registering_02_multiply_initial.png)
+
+Task says: "The code does not execute properly. Try to figure out why." Let's take a closer look at it... Of course! Some tiny detail is missing, and it causes compilation error! Let's put it in place, fix the code, and click `SUBMIT`. Server will process our solution and... after a short while, tests report is presented to you. If you made a mistake and your answer is not correct, don't worry! Just click `TRY AGAIN` and fix your solution.
+
+![retry](./img/registering_03_multiply_wrong_answer.png)
+
+If your answer is correct, you will experience the best little thing in a life of a coder _[citation needed]_: **green tests**! Congratulations!
+
+![success](./img/registering_04_multiply_success.png)
+
+## Your new account
+
+You have solved your first Codewars challenge. Great! Now you are ready to have your account created. Just fill in a couple of details, or use your GitHub account to authenticate on Codewars:
+
+![enlist](./img/registering_05_enlist.png)
+
+Great, your account is registered!
+
+**Important note:** Codewars will send you an email to confirm your email account. Go to your inbox and click on confirmation link in the message titled `"Confirm your account... Dojo Access Link"`, received from Codewars. Until you do this, you will be allowed to train and solve tasks, but some actions will be restricted or unavaliable for you. For example, you won't be able to create discourse posts. If you cannot find the email, check `Spam` folder of your inbox! If you do not confirm your account right now, you can always ask Codewars to resend you the confirmation email by clicking dedicated button in your account panel (we will get to your account panel soon).
+
+## Next steps
+
+You have successfully created your account and registered it with Codewars. You are strongly recommended to [set up](./setting-up/) your account and training routine, but if you wish, you can go straight to your [dashboard](./my-profile/) or even find your [next challenges to solve](./finding-kata/).
```

Comment:
```suggestion
You have successfully created your account and registered it with Codewars. You are strongly recommended to [set up](/getting-started/setting-up/) your account and training routine, but if you wish, you can go straight to your [dashboard](./my-profile/) or even find your [next challenges to solve](/getting-started/finding-kata/).
```

Where should `./my-profile` go?

Reply:
I restored link to `dashboard` in this place, idea of `my-profile` needs more thought.